src/AddTodoForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import InputWithLabel from './InputWithLabel'
+
+function AddTodoForm(props) {
+	const { onAddTodo } = props
```

Comment:
Is it necessary to destructure props with this additional line of code?

Reply:
I guess no though it was on the text  book that was describing how it works. I will clean up!

Reply:
Nice job taking direction from the text and understanding how to apply it but now let's focus on maintaining a consistent coding pattern. e.g. your other components destructure props in the component signature while this one doesn't. You've already set a pattern to follow. Also, it's more common to destructure in the component signature than creating a oneliner. That type of coding pattern actually screams "junior"! :)